bin/midi.ml

```diff
@@ -52,6 +52,7 @@ let write_output { Device.device; _ } msg =
   Portmidi.write_output device msg |> handle_error
 
 module Scale = struct
+  type noteAndVolume = { note : char; vol : char }
```

Comment:
Naming nitpick:

In OCaml, for type names, we tend to favour the _lower camel case_ naming convention. Here it would be `note_and_volume` instead.

It is also likely we'd like to extend this type further, so maybe a more general name would be better, like `note_data`.